app/trezor-usb-permissions.html

```diff
@@ -1,6 +1,8 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
+    <script src="./snow.js" type="text/javascript"  charset="utf-8"></script>
+    <script src="./use-snow.js" type="text/javascript" charset="utf-8"></script>
```

Comment:
I'm not sure how the Trezor integration uses this page. We'll want to test with Trezor to ensure this hasn't broken anything.

Reply:
https://github.com/MetaMask/metamask-extension/pull/20471#pullrequestreview-1583103931